themes/origins/assets/global.js

```diff
@@ -59,3 +59,41 @@ function formatCurrency(amount) {
     ? `${CURRENCY_SYMBOL} ${formattedValue}`
     : `${formattedValue} ${CURRENCY_SYMBOL}`;
 }
+
+relativeTime = (date) => {
+  const diffInSeconds = Math.floor((new Date() - date) / 1000);
+
+  const SECOND = 1;
+  const MINUTE = 60 * SECOND;
+  const HOUR = 60 * MINUTE;
+  const DAY = 24 * HOUR;
+  const WEEK = 7 * DAY;
+  const MONTH = 30 * DAY;
+  const YEAR = 365 * DAY;
+
+  const units = [
+    ["year", YEAR],
+    ["month", MONTH],
+    ["week", WEEK],
+    ["day", DAY],
+    ["hour", HOUR],
+    ["minute", MINUTE],
+    ["second", SECOND],
+  ];
+
+  for (const [unit, seconds] of units) {
+    const interval = Math.floor(diffInSeconds / seconds);
+    if (interval >= 1)
+      return new Intl.RelativeTimeFormat(CUSTOMER_LOCALE, { numeric: "auto" }).format(
+        -interval,
+        unit,
+      );
+  }
+
+  const JUST_NOW = new Intl.RelativeTimeFormat(CUSTOMER_LOCALE, { numeric: "auto" }).format(
```

Comment:
non literal constants should be named with camelCase
```suggestion
  const justNow = new Intl.RelativeTimeFormat(CUSTOMER_LOCALE, { numeric: "auto" }).format(
```

Nitpick, maybe "relativeTime" is a better var name?

Reply:
Here's how we've done it in countdown timer
https://github.com/youcan-shop/youcan-themes/blob/55d0dd9a4cad2a52d3775ed1b76f63b9443fdab1/themes/origins/assets/component-countdown-timer.js#L2